tox.ini

```
@@ -0,0 +1,52 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+[tox]
+skipsdist=True
+skip_missing_interpreters = True
+envlist = lint, unit
+
+[vars]
+src_path = {toxinidir}/src/
+unit_test_path = {toxinidir}/tests/unit/
+lib_path = {toxinidir}/lib/
```

This is just for the check below to work. Feel add the path in the command, it doesn't actually matter.
```suggestion
lib_path = {toxinidir}/lib/charms/kubernetes_charm_libraries/v0
```